surface-model/surface-model_ontology.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
```

Comment:
surface-model_instance.json file is missing.

Reply:
done!